app/models/patient.rb

```diff
@@ -229,6 +229,17 @@ def all_versions(include_fulfillment)
     all_versions.sort_by(&:created_at).reverse
   end
 
+  def as_json
+    super.merge(
+      status: status,
+      last_menstrual_period_at_appt_weeks: last_menstrual_period_at_appt_weeks,
+      last_menstrual_period_at_appt_days: last_menstrual_period_at_appt_days,
+      last_menstrual_period_now_weeks: last_menstrual_period_now_weeks,
+      last_menstrual_period_now_days: last_menstrual_period_now_days,
+      primary_phone_display: primary_phone_display
+    )
+  end
+
```

Comment:
Here again a comment explaining when/why this method is used will really help me in a couple years when I go tracing what's happening!